projects/WTI-API/src/main/config/ServerInit.java

```diff
@@ -29,11 +29,15 @@
  */
 public class ServerInit {
 
+    private final String WTI_PUBLIC_IP_OVERRIDE_KEY = "wtiOverridePublicIP";
```

Comment:
seems this should placed in a sample pc2v9.ini

Reply:
It probably should be.  Unfortunately there are NO samples currently under WTI, so ALL pc2v9.ini attributes need to be added to such a thing.  I'm going to rule that outside the scope of this Issue.